docs/mkdocs.yml

```diff
@@ -22,10 +22,19 @@ nav:
   - index.md
   - install.md
   - analyst.md
-  - developer.md
+  - Developer:
+    - developer/developer.md
+    - developer/dev_environment_setup.md
+    - Architectural Decision Records (ADR):
+       - developer/adr/README.md
+       - developer/adr/0001-pipenv-for-package-and-dependency-management.md
+       - developer/adr/0002-pytest-for-testing.md
+       - developer/adr/0003-http-api.md
+       - developer/adr/0004-http-framework.md
+       - developer/adr/0005-IPC-methods.md
+       - developer/adr/0006-JWTs.md
```

Comment:
Is it possible to add all files in `developer/adr/` here without having to specify each file individually? (I have a feeling the answer is no, but it would be convenient to be able to add new ADRs without having to remember to also add them to the nav).

Reply:
No cake + eating, sadly. You can manually dictate the nav, or you can let it be autogenerated. Can't combine the two.